### DISCUSSION TITLE
Nextflow on SGE

### DISCUSSION THREAD

**Discussion by tanyasarkjain** [nextflow-io/nextflow#5247] [Q&A]
Im trying to run a .nf script on my SGE cluster. I've created a folder with the bin folder that contains the nextflow executable, and added the path to my bashrc file . running the executable directly works fine, but when creating a config file in the directory with my script, and setting executor = 'sge' in the file, im finding  the job won't run - gets stuck on 

<img width="609" alt="Screenshot 2024-08-21 at 5 13 46 PM" src="https://github.com/user-attachments/assets/55195e8d-d035-40ff-8679-cc79afc40780">

**Comment by pditommaso** [comment]
I'd suggest to check the status of SGE jobs in your queue

**Reply by tanyasarkjain** [reply]
It doesn't seem to be getting submitted to the queue at all
